### PR TITLE
Add API functions

### PR DESCRIPTION
**Pull request by Eiphoria**

### CODE REVIEW

File: pidorbot/src/db.py [hunk 1]
```diff
@@ -24,12 +24,25 @@ class User:
     username: str
     selected_count: Optional[int]
 
+    def to_dict(self):
```
Comment (Drozd0f) [inline pidorbot/src/db.py:27]: Add type hint for this method (what that return)

File: pidorbot/src/db.py [hunk 2]
```diff
@@ -24,12 +24,25 @@ class User:
...
+            'selected_count': self.selected_count
+        }
+
 
 @dataclass
 class UsersStat:
     users: List[User]
     in_game_cnt: int
 
+    def to_dict(self):
```
Comment (Drozd0f) [inline pidorbot/src/db.py:40]: Similarly need type hint

File: pidorbot/src/db.py [hunk 3]
```diff
@@ -75,14 +88,13 @@ def remove_database():
 
 
 def result_to_stat(result, chat_id):
```
Comment (Drozd0f) [inline pidorbot/src/db.py:90]: You can use `Optional` to type hint what this function will return

File: pidorbot/src/db.py [hunk 4]
```diff
@@ -75,14 +88,13 @@ def remove_database():
...
-    users = []
-    if result is not None:
+    if result:
         count = get_registered_count(chat_id)
         return UsersStat(
             [User(*row) for row in result],
             count
         )
-    return UsersStat(users, 0)
+    return None
```
Comment (Drozd0f) [inline pidorbot/src/db.py:97]: The function always returns `None`. I think it's no need

**Review by Drozd0f — CHANGES_REQUESTED** [review]

**Comment by qerdcv** [comment]
https://user-images.githubusercontent.com/52095526/178919040-6141d1b1-e840-4a2e-8cb5-91238cfb8048.mp4

**Pull request merged**